python/src/trezorlib/cli/settings.py

```diff
@@ -197,8 +198,11 @@ def safety_checks(client, level):
 
     Set to "strict" to get the full Trezor security.
 
-    Set to "prompt" if you want to be able to allow potentially unsafe actions, such as
-    mismatching coin keys or extreme fees.
+    Set to "prompt-always" if you want to be able to allow potentially unsafe actions,
+    such as mismatching coin keys or extreme fees.
+
+    Set to "prompt-temporarily" to allow potentially unsafe actions until Trezor is
+    unplugged.
```

Comment:
please call the temporary option just `prompt`, and move the description up a paragraph.

that way we indicate that "prompt temporarily" is the recommended thing and "prompt always" is something special

Reply:
What about `trezorctl set safety-checks [--always] [strict|prompt]`? That would make it even more special, though then we'd have to handle `--always strict`.

Reply:
That would also work, I was considering that too. The advantage of doing it via a separate keyword is that we retain mapping to protobuf values. But that's not that important. Do whichever you like better.

Also please add something like "(default setting)" to the "strict" option.

Reply:
Fixed.